README.md

```diff
@@ -261,15 +266,6 @@ decided to change the name of the project to `ComplianceAsCode`, in order to avo
 We envision that the future will be format-agnostic. That's why opted for an
 abstraction instead of using XCCDF for the input format.
 
-## Further reading
-
-The SSG homepage is [https://www.open-scap.org/security-policies/scap-security-guide/](https://www.open-scap.org/security-policies/scap-security-guide/).
```

Comment:
I believe keeping the link to the web page is still relevant. I think you can transform this section into a web site dedicated section for example.

Reply:
That web page is very outdated, it doesn't even mention the rename that our project did long time ago, frequently mentions RHEL 6 and it doesn't add any useful information on top of readthedocs or this readme.

Reply:
I agree that Open-SCAP.org page is very outdated, leaving it out is not too big of an issue.